README.md

```diff
@@ -147,6 +147,22 @@ $ export JDBC_DATABASE_URL="jdbc:postgresql://localhost:5432/pets_test?user=[use
 $ sbt test
 ```
 
+## Deployment
+
+### Heroku ENV Variables
+
+The follow are required to deploy the pet service to Heroku.
```

Comment:
Minor thing, but I misread this as "you need these env vars locally in order to successfully deploy code to heroku". Maybe rephrase to something like "The follow environment variables must be present in Heroku for the app to run" or something. Or honestly, just "In Heroku, set the following 'Config Vars'"

Reply:
Solid feedback. I'll clean that up.